Beam/resources/assets/js/remplib.js

```diff
@@ -119,6 +132,19 @@ remplib = typeof(remplib) === 'undefined' ? {} : remplib;
             window.addEventListener("campaign_showtime", this.syncOverridableFields);
             window.addEventListener("campaign_showtime", this.syncFlags);
             window.addEventListener("beam_event", this.incrementSegmentRulesCache);
+
+            if (this.config.tracker.readingProgress.enabled) {
+
+                setInterval(function () {
+                    this.sendTrackedProgress()
+                }, (this.trackedProgressInterval * 1000));
```

Comment:
The interval is ignoring `readingProgress.interval` configuration option.

Reply:
Meaning that the `init()` function never sets it to anything else and it always remain at default (5 seconds)